README.md

```diff
@@ -102,6 +102,29 @@ Array<{
 };
 ```
 
+### `/api/getPoolInfo`
+
+#### Input
+
+```js
+{
+  poolMetaDataHashes: Array<string> //hash ids of pool meta data.
+};
+```
+
+#### Output
+
+```js
+{
+  [poolMetaDataHash: string]: {| name: string,
```

Comment:
I would prefer if the return type was something like this

```
type RemotePoolInfo = {|
  +owner: string, // I think we don't need this in the backend response since it's specified in the key
  +pledge_address: string,

  // from pool metadata (off chain)
  +name?: string,
  +description?: string,
  +ticker?: string,
  +homepage?: string,
|};

{|
  +info: RemotePoolInfo,
  +history: Array<{|
    epoch: number,
    slot: number,
    tx_ordinal: number,
    cert_ordinal: number,
    payload: any, // TODO: how to store this since different networks have different cert types
  |}>,
|};
```

The main important thing is that we want the `history` list. You can make it an empty array for now, but later it should be a list of all the `PoolRegistration` and `PoolRetirement` that match the pool id

Reply:
Noted.  the history bit will take me a little bit of work to do.  this is essentially just a proxy to smash.  the type I've returned is exactly what smash gives me, no change.  I'll need to populate these fields from the postgresql database.